docs/requirements-doc.txt

```diff
@@ -1,3 +1,6 @@
+# install current modin checkout to bring all required dependencies
+.[all]
```

Comment:
Because right now it fails to run, as current master installs pandas 1.4.4 but in this PR we require 1.5.
Besides, it looks really weird to install dependencies etc from some _other_ checkout rather than current.

Maybe I should stop being lazy and extract that change in its separate PR (and fix the documentation you pointed out)... thoughts?

Reply:
> Maybe I should stop being lazy and extract that change in its separate PR (and fix the documentation you pointed out)... thoughts?

This is fine by me with changing the documentation in this PR.

Reply:
docs updated